project/src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,19 @@
+import {Navigate} from 'react-router-dom';
+import {AppRoute, AuthorizationStatus} from '../../constants';
+
+type PrivateRouteProps = {
+  authorizationStatus: AuthorizationStatus;
+  children: JSX.Element;
+}
+
+const PrivateRoute = (props: PrivateRouteProps): JSX.Element => {
```

Comment:
Пропсы можно сразу достать и лучше придерживаться такого подхода далее
```suggestion
const PrivateRoute = ({authorizationStatus, children}: PrivateRouteProps): JSX.Element => {
```